formats/src/main/kotlin/com/jakewharton/diffuse/format/Signatures.kt

```diff
@@ -50,5 +54,11 @@ data class Signatures(
         closeable?.close()
       }
     }
+
+    @Suppress("UNCHECKED_CAST")
+    private val Result.v4SchemeSigners: List<V4SchemeSignerInfo>
+      get() = this::class.java
+        .getDeclaredMethod("getV4SchemeSigners").apply { isAccessible = true }
```

Comment:
It's private.

```java
  private List<V4SchemeSignerInfo> getV4SchemeSigners() {
      return mV4SchemeSigners;
  }
```

Reply:
Might be a bug of AGP?

Reply:
Find it.
https://issuetracker.google.com/issues/173422058

Reply:
Two years old! Oh my.

I'll try to submit a CL to change it, I guess.

For now we can move forward with the reflection 😬 .

Reply:
This is fixed in AOSP master but that is only for the platform and not the tools. The tools team is aware there's a change that needs pulled in and are going to attempt it soon. At this point that probably means it will be included in 7.3 at some point soon.

Reply:
Sweet, will look back after AGP 7.3 release.

Reply:
It's in #116 for now.